packages/ui/src/__stories__/Tools/ThemeGenerator/ThemeResult/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { ThemeProvider } from '@emotion/react'
+import styled from '@emotion/styled'
+import { useCallback, useState } from 'react'
+import { Button, Row, Stack, Text } from '../../../../components'
+import type { SCWUITheme } from '../../../../theme'
+import consoleLightTheme from '../../../../theme'
+import { normalize } from '../../../../utils'
+import { CodeIntegration } from './CodeIntegration'
+import { Demo } from './Demo' // For some reason Global doesn't work here this is the workaround I found
+
+// For some reason Global doesn't work here this is the workaround I found
+const Container = styled.div`
+  ${normalize()}
+`
+
+type ThemeResultProps = {
+  theme: SCWUITheme
```

Comment:
Not related to this PR, but shouldn't we rename `SCWUITheme` to something like `UltravioletUITheme`?

Reply:
@QuiiBz Good point, I will add a new type instead to avoid breaking changes

Reply:
https://github.com/scaleway/ultraviolet/pull/2967